docs/posts/wow-analysis/index.qmd

```diff
@@ -0,0 +1,174 @@
+---
+title: "Analsis of World of Warcraft Data"
+author: "Tyler White"
+error: false
+date: "2024-02-29"
+categories:
+  - blog
+  - data engineering
+  - duckdb
+---
+## Introduction
+This dataset has been around for a while, but it’s newer to me. I grew up playing video games, and I always had a great time jumping into Azeroth and whatever world was introduced in later expansions with my friends.
+
+This data was captured by a Horde player throughout 2008, between the release of the Burning Crusade and Wrath of the Lich King expansions. On November 13, 2008, we will see many characters navigating new areas and exceeding the previous max level of 70.
+
+Let’s grab the data and try to answer a few questions.
+
+## Analysis
+
+We’ll determine who logged in the most and who leveled from 70 to 80 the fastest. Also, what types of activities were those players engaging in? Let's get to work.
+
+### Getting Started
+
+Ibis ships with an examples module, and this specific data is included. I’ll use DuckDB here, but this is certainly possible with other backends, and I encourage you to experiment. DuckDB is the default Ibis backend, so it’ll be easy to use with this example.
+
+You can execute pip install `ibis-framework[duckdb,examples]` to work with Ibis and the example. You can also throw “jupyter” in to work in IPython or a notebook interface.
```

Comment:
```suggestion
You can execute `pip install ibis-framework[duckdb,examples]` to work with Ibis and the example data.
```

Mentioning jupyter here like this suggests that `pip install ibis-framework[duckdb,examples,jupyter]` is a thing and since it's not I think that will confuse readers.

Reply:
I removed the "jupyter" part to make it clearer.